pkgs/applications/audio/uhhyou.lv2/default.nix

```
@@ -0,0 +1,75 @@
+{ stdenv, fetchgit, pkgconfig, libjack2, liblo, libGL, libX11, lv2, fftw }:
+
+stdenv.mkDerivation rec {
+  pname = "uhhyou.lv2";
+  version = "unstable-2020-05-27";
+
+  src = fetchgit {
+    url = "https://github.com/ryukau/LV2Plugins";
+    rev = "f20d1c8a624d3a757f820b5c41c70f6f9bb6dc22";
+    deepClone = true;
+    sha256 = "12g9j4rrhkw91sbnqi8xg8wkl5j34m0l7fdwhqa2ny526n52lq7k";
+  };
+
+  buildInputs = [ libjack2 liblo libGL libX11 lv2 fftw ];
```

Again, personal preference:

```suggestion
  buildInputs = [
    libjack2
    liblo
    libGL
    libX11
    lv2
    fftw
  ];
```